package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
Unfortunately, webpack doesn't understand it: https://github.com/webpack/webpack/issues/1979#issuecomment-227236676

I'm prototyping some local resolver code that will add these to my src loader pipeline, but without some kind of indicator, I'll be leaving out react-formal.  Not a big deal, I'm just looking for a way to move forward.